### PR TITLE
Fix Travis deploy [Resolves #493]

### LINKED ISSUE #493: Fix pypi deploy from travis
Since switching to travis-ci.com, the pypi deploy no longer works. I tried re-encrypting the password but that did not work.

### PR DESCRIPTION
**Pull request by thcrock**
- Fixes the travis pypi deploy
- Adds a travis github pages deploy and a manage command to run the
local documentation site

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -15,11 +16,20 @@ class Development(LocalRoot):
...
         '-x', 'db_config_file=database.yaml',
         args.remainder,
     ]
+
+
+@Development.register
+class Docs(Local):
+    """View Triage documentation through local server"""
+    def prepare(self, args):
+        with plumlocal.cwd(plumlocal.cwd / 'docs'):
```
Comment (jesteria) [inline manage.py]: Yeah the `local` wrapper that `argcmdr` provides should probably expose more stuff, like the handy `cwd` context manager; but, it doesn't, currently :man_shrugging:

Comment (jesteria) [inline manage.py]: I think you want:

```suggestion
        with plumlocal.cwd(ROOT_PATH / 'docs'):
```

File: manage.py [hunk 2]
```diff
@@ -15,11 +16,20 @@ class Development(LocalRoot):
 @Development.register
 @local('remainder', metavar='alembic arguments', nargs=argparse.REMAINDER)
 def alembic(context, args):
-    """Configuration wrapper to use the Alembic schema migrations library for Triage development"""
+    """Configuration wrapper to use the Alembic schema migrations library for Triage development.
+    Try alembic -h to see description of all available subcommands"""
```
Comment (jesteria) [inline manage.py]: Not a bad note to include; but, might be at least as useful to explain that these should also work:

```
manage alembic - -h
manage alembic -- -h
```

It's an annoying thing to get around `argparse` seeing the help flag and immediately exiting, (but also not uncommon with other CLIs, even Git's).

Comment (thcrock) [reply manage.py]: Ohhhh I was wondering if there was a way around that

**Comment by jesteria** [comment]
Yeah 🤷

That pops up from time to time, especially when passing arguments to
subprocesses. (I even recently saw it in the `git stash` command docs.)

**Pull request merged**